modules/rhdh-plugins-reference/proc-create-plugin.adoc

```diff
@@ -0,0 +1,38 @@
+[id="proc-create-plugin"]
```

Comment:
I thought this would be removed from here and added into a temporary Google Dic ?

Reply:
This file has been commented out in the index file, so it won't appear in the documentation until we finalize the content and add it to this file. I haven't removed it entirely because we plan to add this section in the future, and its placement will remain as is.
https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/199/files#diff-d93141c559e24aefd522d9ef3e2772697e45ac1a759fa3598f02888bf9c13a82R17